src/Tizen.MachineLearning.Inference/Tizen.MachineLearning.Inference/MlInformation.cs

```diff
@@ -0,0 +1,266 @@
+                throw NNStreamerExceptionFactory.CreateException(NNStreamerError.InvalidParameter, "The property key is invalid");
+
+            if (string.IsNullOrEmpty(value))
+                throw NNStreamerExceptionFactory.CreateException(NNStreamerError.InvalidParameter, "The property value is invalid");
+
+            NNStreamerError ret = NNStreamerError.InvalidParameter;
+
+            switch(_type)
+            {
+                case InfoType.Option:
+                    IntPtr valuePtr = Interop.Util.StringToIntPtr(value);
+                    ret = Interop.Util.SetOptionValue(_handle, key, valuePtr, null);
+                    break;
+                case InfoType.Information:
+                    ret = NNStreamerError.NotSupported;
+                    Log.Error(NNStreamer.TAG, "InfoType Information does not support set value");
+                    break;
+            }
+
+            NNStreamer.CheckException(ret, "Failed to set string value");
+        }
+
+        /// <summary>
+        /// Sets a new key-value in mlInformation instance.
+        /// </summary>
+        /// <param name="key">The key to be set.</param>
+        /// <param name="value">The value to be set.</param>
+        /// <feature>http://tizen.org/feature/machine_learning.inference</feature>
+        /// <exception cref="ArgumentException">Thrown when the method failed due to an invalid parameter.</exception>
+        /// <exception cref="NotSupportedException">Thrown when the feature is not supported.</exception>
+        /// <since_tizen> 12 </since_tizen>
+        public void SetNumber(string key, int value)
+        {
+            if (string.IsNullOrEmpty(key))
+                throw NNStreamerExceptionFactory.CreateException(NNStreamerError.InvalidParameter, "The property key is invalid");
+
+            NNStreamerError ret = NNStreamerError.InvalidParameter;
+
+            switch(_type)
+            {
+                case InfoType.Option:
+                    ret = Interop.Util.SetOptionValue(_handle, key, Interop.Util.IntToIntPtr(value), null);
+                    break;
+                case InfoType.Information:
+                    ret = NNStreamerError.NotSupported;
+                    Log.Error(NNStreamer.TAG, "InfoType Iniformation does not support set value");
+                    break;
+            }
+
+            NNStreamer.CheckException(ret, "Failed to set string value");
+        }
+
+        private IntPtr GetInformationHandle(string key)
+        {
+            if (string.IsNullOrEmpty(key))
+                throw NNStreamerExceptionFactory.CreateException(NNStreamerError.InvalidParameter, "The property key is invalid");
+
+            IntPtr value = IntPtr.Zero;
+            NNStreamerError ret = NNStreamerError.InvalidParameter;
+
+            switch(_type)
+            {
+                case InfoType.Option:
+                    ret = Interop.Util.GetOptionValue(_handle, key, out value);
```

Comment:
Q: need conversion 'key' to intptr?

Reply:
The `key` value is only string type now, but as we discuss earlier I updated it to `IntPtr`
Thank you :)